config/routes.rb

```diff
@@ -1,7 +1,8 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
   root "user#index"
```

Comment:
- [ ] You have a routing error. This would be

  ```
  users#index
  ```

  ![image](https://github.com/SaveryIV/blog-app-on-Rails/assets/54848363/f4fea022-090f-4769-bd62-1d8f8fba3e59)